### PR TITLE
Com tools updates

### PR DESCRIPTION
**Pull request by erichhuebner**
com_tools updates to include SImtool options.

### CODE REVIEW

File: core/components/com_tools/site/controllers/pipeline.php [hunk 1]
```diff
@@ -1010,8 +1025,18 @@ public function saveTask()
 			$txt->set('repohost', $tool['repohost']);
 		}
 
-		$ptype = (empty($tool['publishType']) || $tool['publishType'] == 'standard') ? 'standard': 'weber=';
+		//$ptype = (empty($tool['publishType']) || $tool['publishType'] == 'standard') ? 'standard': 'weber=';
```
Comment (damionjunk) [inline core/components/com_tools/site/controllers/pipeline.php:1028]: deletemeplz

File: core/components/com_tools/site/views/pipeline/tmpl/edit.php [hunk 2]
```diff
@@ -224,19 +252,21 @@
...
+							</label>
+						</div>
+					<?php } ?>
 				</fieldset>
 	
 				<?php if ($this->config->get('github', 1)) { ?>
 					<div id="gitExternalExplanation" class="explaination">
 						<?php if (file_exists('/usr/bin/addrepo.sh')) { ?>
-							<p><?php echo Lang::txt('You can optionally provide a URL for your Git repository containing the tool code, and we will grab code from this repository when installing your tool.  Public repositories should be specified as https://git@github.com/[reponame] while private repositories should be specified as ssh://git@github.com/[reponame]'); ?></p>
+							<p><?php echo Lang::txt('You can optionally provide a URL for your Git repository containing the tool code, and we will grab code from this repository when installing your tool.  Public repositories should be specified as https://github.com/[reponame] while private repositories should be specified as ssh://github.com/[reponame]'); ?></p>
```
Comment (damionjunk) [inline core/components/com_tools/site/views/pipeline/tmpl/edit.php:269]: verify the ssh:// lack of username requirement

File: core/components/com_tools/site/views/pipeline/tmpl/edit.php [hunk 3]
```diff
@@ -224,19 +252,21 @@
...
-						</label>
-					</div>
+
+					<?php if ($this->config->get('github', 1)) { ?>
+						<div class="form-group form-check">
+							<label for="tool_repohost_gitexternal" class="form-check-label">
+								<input type="radio" name="tool[repohost]" id="tool_repohost_gitexternal" value="gitExternal" <?php if ($this->defaults['repohost'] == 'gitExternal') { echo 'checked="checked"'; } ?> class="option form-check-input" />
+								<?php echo Lang::txt('Host GIT repository on GitHUB'); ?>
+							</label>
+						</div>
```
Comment (damionjunk) [inline core/components/com_tools/site/views/pipeline/tmpl/edit.php:262]: language string ref

File: core/components/com_tools/site/views/pipeline/tmpl/status.php [hunk 4]
```diff
@@ -397,6 +397,7 @@
 					<?php if (!empty($rappture_url)) { ?>
 						<li><?php echo Lang::txt('COM_TOOLS_LEARN_MORE'); ?> <?php echo Lang::txt('COM_TOOLS_ABOUT'); ?> <?php echo Lang::txt('COM_TOOLS_THE'); ?> <a href="<?php echo $rappture_url; ?>">Rappture toolkit</a>.</li>
 					<?php } ?>
+						<li>Learn about nanoHUB’s <a href="/whypublish/whypublishdevenv">software development environment</a></li>
```
Comment (damionjunk) [inline core/components/com_tools/site/views/pipeline/tmpl/status.php:400]: nanohub specific?

File: core/components/com_tools/site/views/resource/tmpl/tags.php [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+ * @license   http://opensource.org/licenses/MIT MIT
+ */
+
+// No direct access.
+defined('_HZEXEC_') or die();
+
+$showwarning = ($this->version=='current' or !$this->status['published']) ? 0 : 1;
+
+?>
+	<div class="explaination">
```
Comment (damionjunk) [inline core/components/com_tools/site/views/resource/tmpl/tags.php:39]: check with the css

File: core/components/com_tools/site/views/resource/tmpl/tags.php [hunk 6]
```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * HUBzero CMS
+ *
+ * Copyright 2005-2015 HUBzero Foundation, LLC.
```
Comment (damionjunk) [inline core/components/com_tools/site/views/resource/tmpl/tags.php:5]: license dates and ownership

File: core/components/com_tools/site/views/pipeline/tmpl/edit.php (hunk 2 above)
Comment (erichhuebner) [reply core/components/com_tools/site/views/pipeline/tmpl/edit.php:269]: git username is required.

File: core/components/com_tools/site/views/pipeline/tmpl/status.php (hunk 4 above)
Comment (erichhuebner) [reply core/components/com_tools/site/views/pipeline/tmpl/status.php:400]: Yes, removed.

File: core/components/com_tools/site/views/resource/tmpl/tags.php (hunk 5 above)
Comment (erichhuebner) [reply core/components/com_tools/site/views/resource/tmpl/tags.php:39]: Yes...it's misspelled everywhere...

**Comment by rw2** [comment]
Closed as duplicate of another PR containing the "real" code.

**Pull request closed without merging**